### PR TITLE
Optimized Getallcontents

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

https://github.com/tgstation/TerraGov-Marine-Corps/pull/5503
image

## Changelog
:cl:
code: Optimized Getallcontents
/:cl:

### CODE REVIEW

File: code/__HELPERS/unsorted.dm [hunk 1]
```diff
@@ -414,30 +414,25 @@ Turf and target are separate in case you want to teleport some distance from a t
...
-		. = list()
-		var/i = 0
-		while(i < length(processing_list))
-			var/atom/A = processing_list[++i]
-			//Byond does not allow things to be in multiple contents, or double parent-child hierarchies, so only += is needed
-			//This is also why we don't need to check against assembled as we go along
-			if (!(A.flags_1 & ignore_flag_1))
-				processing_list += A.contents
-				if(istype(A,T))
-					. += A
```
Comment (LemonInTheDark) [inline code/__HELPERS/unsorted.dm:433]: We have extra behavior here that you have overwritten, please address this

File: code/__HELPERS/unsorted.dm [hunk 2]
```diff
@@ -414,30 +414,25 @@ Turf and target are separate in case you want to teleport some distance from a t
 	var/y = min(world.maxy, max(1, A.y + dy))
 	return locate(x,y,A.z)
 
-/*
-	Gets all contents of contents and returns them all in a list.
-*/
```
Comment (LemonInTheDark) [inline code/__HELPERS/unsorted.dm:419]: Why did you delete the documentation?

File: code/__HELPERS/unsorted.dm (hunk 1 above)
Comment (TiviPlus) [reply code/__HELPERS/unsorted.dm:433]: Ignore flag was competely unused

File: code/__HELPERS/unsorted.dm (hunk 2 above)
Comment (LemonInTheDark) [reply code/__HELPERS/unsorted.dm:419]: ah excuse me

File: code/__HELPERS/unsorted.dm (hunk 1 above)
Comment (LemonInTheDark) [reply code/__HELPERS/unsorted.dm:433]: lol
